pom.xml

```diff
@@ -265,14 +265,14 @@
                      Format is "<preferred license name> :: <file name>|<license name>|<license name>|..."
                     -->
                     <licenseMerges>
-                        <licenseMerge>Apache Software License 2.0 :: apache_software_license_2.0.txt|Apache Software License, Version 2.0|The Apache Software License, Version 2.0|Apache License, Version 2.0|Apache License 2.0|Apache 2|Apache 2.0|Apache-2.0</licenseMerge>
-                        <licenseMerge>MIT License :: mit_license.txt|MIT License|The MIT License|MIT X11 License|MIT</licenseMerge>
+                        <licenseMerge>Apache Software License 2.0 :: apache_software_license_2.0.txt|Apache Software License, Version 2.0|The Apache Software License, Version 2.0|Apache License, Version 2.0|Apache License 2.0|Apache 2|Apache 2.0|Apache-2.0|The Apache License, Version 2.0</licenseMerge>
+                        <licenseMerge>MIT License :: mit_license.html|MIT License|The MIT License|MIT X11 License|MIT</licenseMerge>
                         <licenseMerge>Eclipse Public License 1.0 :: eclipse_public_license_1.0.html|Eclipse Public License 1.0|The Eclipse Public License 1.0|Eclipse Public License - v 1.0|Eclipse Distribution License - v 1.0</licenseMerge>
                         <licenseMerge>Eclipse Public License 2.0 :: eclipse_public_license_2.0.html|Eclipse Public License v2.0</licenseMerge>
                         <licenseMerge>BSD License 3 :: bsd_license_3.html|BSD Licence 3</licenseMerge>
                         <licenseMerge>GNU Lesser General Public License 2.1 :: gnu_lgpl_2.1.html|GNU Lesser General Public License</licenseMerge>
                         <licenseMerge>Mozilla Public License 2.0 or Eclipse Public License 1.0 :: mpl_2.0_or_epl_1.0.html|MPL 2.0 or EPL 1.0</licenseMerge>
-                        <licenseMerge>Common Development and Distribution License (CDDL) 1.1 :: cddl_gpl_1.1.txt|CDDL 1.1|cddl+gpl_1.1.html|CDDL+GPL License</licenseMerge>
+                        <licenseMerge>Common Development and Distribution License (CDDL) 1.1 :: cddl_gpl_1.1.html|CDDL 1.1|cddl+gpl_1.1.html|CDDL+GPL License</licenseMerge>
```

Comment:
Intentional licence format change?

Reply:
The dependency that provided the .txt MIT license doesn't exist, so a clean generation of the licenses won't provide it.
The .html one is still there